ext/pdo_pgsql/pgsql_driver.stub.php

```diff
@@ -33,4 +33,7 @@ public function pgsqlGetNotify(int $fetchMode = PDO::FETCH_DEFAULT, int $timeout
 
     /** @tentative-return-type */
     public function pgsqlGetPid(): int {}
+
+    /** @tentative-return-type */
+    public function pgsqlSetNoticeCallback(?callable $callback): bool {}
```

Comment:
This function either succeeds or fails and throws. So the boolean return type is not necessary.

```suggestion
    public function pgsqlSetNoticeCallback(?callable $callback): void {}
```